src/paho/mqtt/__init__.py

```diff
@@ -1,4 +1,10 @@
-__version__ = "1.2.2"
+__version__ = "1.3.0.dev0"
```

Comment:
The issue about dependency is only true if the module that contains version (and which is imported from setup.py) needs a dependency.
Here we import an "empty" __init__.py that contains only the version, which is fine.
That being said, it could indeed only be present in setup.py and being queried thought pkg_resource. This seems more Pythonic.